modules/Bio/EnsEMBL/Compara/PipeConfig/DatachecksForRelease_conf.pm

```diff
@@ -0,0 +1,374 @@
+                datacheck_groups => [
+                    'compara',
+                ],
+                datacheck_types  => [
+                    'critical',
+                ],
+            },
+            -flow_into         => {
+                '2->A' => 
+                    WHEN(
+                        '#parallelize_datachecks#' => [ 'DataCheckFactory' ],
+                        ELSE [ 'RunDataChecks' ]
+                    ),
+                'A->1' => [ 'DataCheckResults' ],
+            },
+        },
+
+        {
+            -logic_name        => 'RunDataChecks',
+            -module            => 'Bio::EnsEMBL::Compara::RunnableDB::RunDataChecks',
+            -analysis_capacity => 10,
+            -max_retry_count   => 0,
+            -rc_name           => '2Gb_job',
+            -flow_into         => {
+                '1'  => [ 'StoreResults' ],
+                '-1' => [ 'RunDataChecks_High_mem' ]
+            },
+        },
+
+        {
+            -logic_name        => 'RunDataChecks_High_mem',  
+            -module            => 'Bio::EnsEMBL::Compara::RunnableDB::RunDataChecks',
+            -analysis_capacity => 10,
+            -max_retry_count   => 0,
+            -rc_name           => '2Gb_job',
+            -flow_into         => {
+                '1' => [ 'StoreResults' ],
+            },
+        },
+
+        {
+            -logic_name        => 'DataCheckFactory',
+            -module            => 'Bio::EnsEMBL::DataCheck::Pipeline::DataCheckFactory',
+            -analysis_capacity => 10,
+            -max_retry_count   => 0,
+            -flow_into         => {
+                '2->A' => [ 'DataCheckFan' ],
+                'A->1' => [ 'DataCheckFunnel' ],
+            },
+        },
+
+        {
+            -logic_name        => 'DataCheckFan',
+            -module            => 'Bio::EnsEMBL::Compara::RunnableDB::DataCheckFan',
+            -analysis_capacity => 100,
+            -max_retry_count   => 0,
+            -rc_name           => '500Mb_job',
+            -flow_into         => {
+                '1'  => [ '?accu_name=results&accu_address=[]' ],
+                '-1' => [ 'DataCheckFan_High_mem' ],
+                '2'  => [ 'JiraTicketCreation' ],
+            },
+        },
+
+        {
+            -logic_name        => 'DataCheckFan_High_mem',
+            -module            => 'Bio::EnsEMBL::Compara::RunnableDB::DataCheckFan',
+            -analysis_capacity => 100,
+            -max_retry_count   => 0,
+            -rc_name           => '2Gb_job',
+            -flow_into         => {
+                '1' => [ '?accu_name=results&accu_address=[]' ],
+                '2' => [ 'JiraTicketCreation' ],
+            },
+        },
+
+        {
+            -logic_name        => 'JiraTicketCreation',
+            -module            => 'Bio::EnsEMBL::Compara::RunnableDB::CreateDCJiraTickets',
+            -parameters        => {
+                division                     => '#division#',
```

Comment:
I'm surprised this works, I have had issues in the past with declarations like this one. It is safer to do:
```suggestion
                division                     => $self->o('division'),
```